types/ember__component/index.d.ts

```diff
@@ -93,3 +94,17 @@ export default class Component extends CoreView.extend(
      */
     willUpdate(): void;
 }
+
+/**
+ * Associate a class with a component manager (an object that is responsible for
+ * coordinating the lifecycle events that occurs when invoking, rendering and
+ * re-rendering a component).
+ *
+ * @param managerFactory a function to create the owner for an object
+ * @param object the object to associate with the componetn manager
+ * @return the same object passed in, now associated with the manager
+ */
+export function setComponentManager<T>(
```

Comment:
Do we need to declare the `capabilities` export as well? As far as I know that's the only documented(?) way to produce a `ComponentCapabilities` object, and without it I'm not sure how authors could get their hands on a value that would be legal to use with `setComponentManager` as declared here

Reply:
Mmmmm, probably. I’m poking at it and I don’t actually know how to expose some of this (it’s not really exposed in a clean way in Ember’s or Glimmer’s internal types, such that it’s not clear what’s intended to be public and what isn’t). Will check with folks who’d know.